sci-calculators/wcalc/wcalc-2.5.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils
+
+DESCRIPTION="A flexible command-line scientific calculator"
+HOMEPAGE="http://w-calc.sourceforge.net/"
+SRC_URI="mirror://sourceforge/w-calc/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+IUSE="readline"
+
+DEPEND="
+	dev-libs/gmp:0=
+	dev-libs/mpfr:0
+	readline? ( sys-libs/readline:0 )"
+RDEPEND="${DEPEND}"
```

Comment:
In general, its more preferable to define the dependencies in `RDEPEND`, and then include `RDEPEND` in `DEPEND`